crypto/ec/ec_lib.c

```
@@ -1559,6 +1566,9 @@ EC_GROUP *EC_GROUP_new_from_params(const OSSL_PARAM params[],
         }
         return group;
     }
+#ifdef FIPS_MODULE
+    return NULL;
```

Should we raise an error here about unsupported explicit parameters?

---

Good point, I'll add it.